tasks/smoke-test/playwright.config.ts

```diff
@@ -3,10 +3,11 @@ import { devices as replayDevices } from '@replayio/playwright'
 
 // See https://playwright.dev/docs/test-configuration#global-configuration
 const config: PlaywrightTestConfig = {
-  timeout: 90_000,
+  timeout: 90_000 * 3,
   expect: {
     timeout: 10 * 1000,
   },
+  workers: 1, // do not run things in parallel
```

Comment:
I could experiment with these settings more, but so far I had to increase the timeout and borrow the strategy used in the Vite PR to get the smoke tests to pass